### PR TITLE
Switch to using Travis CI for Windows / Nightly notifications

### PR DESCRIPTION
**Pull request by omus**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,17 +2,34 @@ language: julia
...
   - linux
+  - windows
+arch:
+  - x64
+  - x86
 julia:
-  - 1.0
-  - 1.1
+  - 1.0  # Latest LTS release
+  - 1    # Latest stable release
```
Comment (mjram0s) [inline .travis.yml:11]: `1` grabs the latest stable release?

Comment (omus) [reply .travis.yml:11]: Yes it does. For our repos we kind of flip/flop between using this and explicitly stating 1.3 or 1.4. I think using 1 is preferable to repos that aren't updated frequently like this one.

**Comment by omus** [comment]
Windows on Julia 1.0 failed. Will check if it is reproducible

**Review by mattBrzezinski — APPROVED** [review]
LGTM, once they CI is happy I think it's good to go!

**Review by mjram0s — APPROVED** [review]

**Comment by omus** [comment]
Failure seems reproducible:
```
binary_ascii: Error During Test at C:\Users\travis\build\invenia\FTPClient.jl\test\non_ssl.jl:205
  Got exception outside of a @test
   :: LibCURL error #56
  Stacktrace:
   [1] macro expansion at C:\Users\travis\build\invenia\FTPClient.jl\src\FTPC.jl:62 [inlined]
   [2] ftp_perform(::ConnContext{Bool}) at C:\Users\travis\build\invenia\FTPClient.jl\src\conn_context.jl:297
   [3] ftp_get(::ConnContext{Bool}, ::String, ::String; mode::FTP_MODE) at C:\Users\travis\build\invenia\FTPClient.jl\src\conn_context.jl:151
   [4] ftp_get(::RequestOptions, ::String, ::String; mode::FTP_MODE, verbose::Bool) at C:\Users\travis\build\invenia\FTPClient.jl\src\request_options.jl:114
   [5] ftp_get at C:\Users\travis\build\invenia\FTPClient.jl\src\request_options.jl:112 [inlined] (repeats 2 times)
   [6] top-level scope at C:\Users\travis\build\invenia\FTPClient.jl\test\non_ssl.jl:229
   [7] top-level scope at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.4\Test\src\Test.jl:1113
   [8] top-level scope at C:\Users\travis\build\invenia\FTPClient.jl\test\non_ssl.jl:207
   [9] include(::String) at .\client.jl:439
   [10] top-level scope at C:\Users\travis\build\invenia\FTPClient.jl\test\runtests.jl:41
   [11] top-level scope at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.4\Test\src\Test.jl:1113
   [12] top-level scope at C:\Users\travis\build\invenia\FTPClient.jl\test\runtests.jl:40
   [13] top-level scope at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.4\Test\src\Test.jl:1113
   [14] top-level scope at C:\Users\travis\build\invenia\FTPClient.jl\test\runtests.jl:29
   [15] include(::String) at .\client.jl:439
   [16] top-level scope at none:6
   [17] eval(::Module, ::Any) at .\boot.jl:331
   [18] exec_options(::Base.JLOptions) at .\client.jl:264
   [19] _start() at .\client.jl:484
```
Windows 64-bit, Julia 1.4.1

**Comment by omus** [comment]
I think this was just a random error. Hopefully we can uncover the root cause with nightlies.

**Pull request merged**